src/server/models/Unit.js

```diff
@@ -88,21 +88,21 @@ class Unit {
 		// The table contains units' data. 
 		// Each row contains: name, identifier, typeOfUnit, suffix, displayable, preferredDisplay.
 		const standardUnits = [
-			['kWh', 'kWh', Unit.unitType.UNIT, '', Unit.displayableType.ALL, true],
-			['MJ', 'MegaJoules', Unit.unitType.UNIT, '', Unit.displayableType.ADMIN, false],
-			['BTU', 'BTU', Unit.unitType.UNIT, '', Unit.displayableType.ALL, true],
-			['M3_gas', 'cubic meters of gas', Unit.unitType.UNIT, '', Unit.displayableType.ALL, false],
-			['kg', 'kg', Unit.unitType.UNIT, '', Unit.displayableType.ALL, false],
-			['Metric_ton', 'Metric ton', Unit.unitType.UNIT, '', Unit.displayableType.ALL, false],
-			['Fahrenheit', 'Fahrenheit', Unit.unitType.UNIT, '', Unit.displayableType.ALL, false],
-			['Celsius', 'Celsius', Unit.unitType.UNIT, '', Unit.displayableType.ALL, false]
+			['kWh', 'kWh', Unit.unitRepresentType.QUANTITY, Unit.unitType.UNIT, '', Unit.displayableType.ALL, true],
+			['MJ', 'MegaJoules', Unit.unitRepresentType.QUANTITY, Unit.unitType.UNIT, '', Unit.displayableType.ADMIN, false],
+			['BTU', 'BTU', Unit.unitRepresentType.QUANTITY, Unit.unitType.UNIT, '', Unit.displayableType.ALL, true],
+			['M3_gas', 'cubic meters of gas', Unit.unitRepresentType.QUANTITY, Unit.unitType.UNIT, '', Unit.displayableType.ALL, false],
+			['kg', 'kg', Unit.unitRepresentType.QUANTITY, Unit.unitType.UNIT, '', Unit.displayableType.ALL, false],
+			['Metric_ton', 'Metric ton', Unit.unitRepresentType.QUANTITY, Unit.unitType.UNIT, '', Unit.displayableType.ALL, false],
+			['Fahrenheit', 'Fahrenheit', Unit.unitRepresentType.FLOW, Unit.unitType.UNIT, '', Unit.displayableType.ALL, false],
```

Comment:
Temperature is raw in OED. The flow ones are for ga/hour, kW, etc.

Reply:
I have fixed this in the new commit.